modules/admin/proc-migrating-databases-to-an-external-server.adoc

```diff
@@ -0,0 +1,92 @@
++
+[source,terminal]
+----
+oc port-forward -n <your-namespace> <pgsql-pod-name> <forward-to-port>:<forward-from-port>
+----
+Where:
+* The `<pgsql-pod-name>` variable denotes the name of a PostgreSQL pod with the format `backstage-psql--<_index>`.
+* The `<forward-to-port>` variable denotes the port of your choice to forward PostgreSQL data to.
+* The `<forward-from-port>` variable denotes the local PostgreSQL instance port, such as `5432`.
++
+.Example: Configuring port forwarding
+[source,terminal]
+----
+oc port-forward -n backstage backstage-psql-backstage1-0 15432:5432
+----
+
+. Make a copy of the `db_copy.sh` script and edit the following details based on your configuration:
+
+* `to_host`: The destination host name, for example, `<db-instance-name>.rds.amazonaws.com`.
+* `to_port`: The destination port, such as `5432`.
+* `to_user`: The destination server username, for example, `postgres`.
+* `from_host`: The source host name, such as `127.0.0.1`.
+* `from_port`: The source port number, such as the `<forward-to-port>` variable. 
+* `from_user`: The source server username, for example, `postgres`.
+* `allDB`: The name of databases to import in double quotes separated by spaces, for example, `("backstage_plugin_app" "backstage_plugin_auth" "backstage_plugin_catalog" "backstage_plugin_permission" "backstage_plugin_scaffolder" "backstage_plugin_search")`.
+
+. Create a destination database for copying the data:
++
+[source,terminal]
+----
+/bin/bash TO_PSW=<destination-db-password> /path/to/db_copy.sh <1>
+----
+<1> The `<destination-db-password>` variable denotes the password to connect to the destination database.
++
+[NOTE]
+====
+You can stop port forwarding when the copying of the data is complete. For more information about handling large databases and using the compression tools, see the link:https://www.postgresql.org/docs/current/backup-dump.html#BACKUP-DUMP-LARGE[Handling Large Databases] section on the PostgreSQL website.
+====
+
+. Reconfigure your `Backstage` custom resource (CR). For more information, see link:{LinkAdminGuide}#proc-configuring-postgresql-instance-using-operator_admin-rhdh[Configuring an external PostgreSQL instance using the Operator]. 
+. Check that the following code is present at the end of your `Backstage` CR after reconfiguration:
++
+[source,yaml]
+----
+# ...
+spec:
+  database:
+    enableLocalDb: false 
+  application:
+  # ... 
+    extraFiles:
+      secrets:
+        - name: <crt-secret> 
+          key: postgres-crt.pem # key name as in <crt-secret> Secret
+    extraEnvs:
+      secrets:
+        - name: <cred-secret> 
+# ...        
+----
++
+[NOTE]
+====
+Reconfiguring the `Backstage` CR deletes the corresponding `StatefulSet` and `Pod` objects, but does not delete the `PersistenceVolumeClaim` object. Use the following command to delete the local `PersistenceVolumeClaim` object:
+
+[source,terminal]
+----
+oc -n backstage delete pvc <local-psql-pvc-name>
+----
+====
```

Comment:
Better to add the note "Where `<local-psql-pvc-name>` has format `data-<psql-pod-name>` (see above)"